lib/profiles/TR/CRD.js

```diff
@@ -8,9 +8,13 @@ exports.config = {
 };
 
 const base = require('../base');
-const rules = base.removeRules(
+let rules = base.removeRules(
     require('./CR').rules,
     'sotd.candidate-review-end'
 );
 
+rules = base.insertAfter(rules, 'sotd.supersedable', [
```

Comment:
This is quite messy.
* TR.js has `draft-stability`
* CR.js extends TR.js but removes `draft-stability` from the rules list

and now, CRD.js extends CR.js but re-adds `draft-stability`. This requires some clean up.